webinterface/_base.py

```diff
@@ -35,15 +35,24 @@ def _preface(self):
             """,
             unsafe_allow_html=True,
         )
+        st.image("https://github.com/Proteobench/ProteoBench/blob/Add-logos/webinterface/logos/logo_participants/logos_all_20230926.png?raw=true")
```

Comment:
```suggestion
        st.image("webinterface/logos/logo_participants/logos_all_20230926.png")
```

Depending what the root folder the application should be started from, I would suspect that this is works as well. 

I just saw now: you will need to change the link once it is merged as it points to the `Add-logos` branch. So try to make them all relative paths:)